Source/WebGPU/WebGPU/Device.h

```diff
@@ -112,6 +112,8 @@ class Device : public WGPUDeviceImpl, public ThreadSafeRefCountedAndCanMakeThrea
 
     uint32_t maxBuffersPlusVertexBuffersForVertexStage() const;
     uint32_t vertexBufferIndexForBindGroup(uint32_t groupIndex) const;
+    id<MTLBuffer> newBufferWithBytes(const void *, NSUInteger, MTLResourceOptions) const;
```

Comment:
Style-wise, I think the new hunks have wrong position for the star in pointers

Might also in the future be easier to ingest size_t, as most of callers seem to do casting to NSUInteger.